content/docs/ibm/install-kubeflow.md

```diff
@@ -116,6 +116,61 @@ components, the recommended configuration for a cluster is:
 
     Make sure all the nodes are in `Ready` state. You are now ready to install
     Istio into your cluster.
+    
+## IBM Cloud Block Storage Setup
+[IBM Cloud Block Storage](https://www.ibm.com/cloud/block-storage) provides a fast way to store data and
+satisfy many of the Kubeflow persistent volume requirements such as `fsGroup` out of the box. Therefore,
+we strongly recommend to set up IBM Cloud Block Storage as the default storageclass so that you can
+get the best experience from Kubeflow.
```

Comment:
Explain more, and a line about NFS storage which is an alternate option, but why it might not be conducive

Reply:
Thanks Animesh, I extended this section to explain why File Storage is not sufficient for DEX and Jupyter.